### DISCUSSION TITLE
Is any per user subscription limit forseen?

### DISCUSSION THREAD

**Discussion by skinkie** [halfgaar/FlashMQ#55] [Ideas]
While it is clear FlashMQ mentions plugins to be used for customizing acls, I wonder if any general limitations are forseen such as limit the number of times a single user entity can have an open session.

**Comment by halfgaar** [comment]
The short answer is no. Implementing with a plugin is also not easy, because there is no way to ask how many (offline) sessions there are with a specific username.

**Reply by skinkie** [reply]
Doing it in a firewall also does not sound as the most fun thing to implement :)
